src/components/D3tree.js

```diff
@@ -171,8 +180,17 @@ export default function D3Tree() {
           </div>
         </Modal>
       </div>
-      <div>path: {nodeFile?.fileName}</div>
-      <pre className="code-container">{code}</pre>
+      {nodeFile ? (
+        <>
+          <div className="buttonNav">
+            <div>path: {nodeFile?.fileName}</div>
+            <button type="button" onClick={closeEditor}>
+              X
+            </button>
+          </div>
+          <Codeviewer code={code} />
+        </>
+      ) : null}
```

Comment:
&&연산자를 사용할 수도 있을 것 같습니다.
그리고 중간에 optional chaining은 없어도 될 것 같습니다.
```suggestion
      {nodeFile && (
        <>
          <div className="buttonNav">
            <div>path: {nodeFile.fileName}</div>
            <button type="button" onClick={closeEditor}>
              X
            </button>
          </div>
          <Codeviewer code={code} />
        </>
      )}
```